mw-webapp/src/service/DayReportService.ts

```diff
@@ -70,7 +96,20 @@ export class DayReportService {
     const dayReportRaw = await getDoc(doc(db, PATH_TO_DAY_REPORTS_COLLECTION, uuid));
     const dayReportDTO = documentSnapshotToDTOConverter<DayReportDTO>(dayReportRaw);
 
-    const validatedDayReportDTO = DayReportDTOSchema.parse(dayReportDTO);
+    const jobsDone = dayReportDTO.jobsDoneStringified.map((item) => createModelFromJson<JobDoneDTO>(item));
```

Comment:
Looks like this kind of model will not able to include any custom logic which is available in constructors. That mean that createModelFromJson is not craeting models. It just parse json.

Reply:
Renamed